models/dto/transaction_base.go

```diff
@@ -7,14 +7,15 @@ import (
 )
 
 type TransactionBase struct {
-	Hash         common.Hash
-	TxType       txtype.TransactionType
-	FromStateID  uint32
-	Amount       models.Uint256
-	Fee          models.Uint256
-	Nonce        models.Uint256
-	Signature    models.Signature
-	ReceiveTime  *models.Timestamp
+	Hash        common.Hash
+	TxType      txtype.TransactionType
+	FromStateID uint32
+	Amount      models.Uint256
+	Fee         models.Uint256
+	Nonce       models.Uint256
+	Signature   models.Signature
+	ReceiveTime *models.Timestamp
+	// TODO: is it safe to change this?
 	CommitmentID *CommitmentID
```

Comment:
Does the mobile app currently expect a specific result, is it safe to add `IndexInCommitment`?